src/test/groovy/com/streamr/broker/stats/LoggedStatsSpec.groovy

```diff
@@ -60,9 +60,9 @@ class LoggedStatsSpec extends Specification {
 		recordedLogs == [
 			"Statistics logger started. Logging interval is 1 sec(s).",
 			"\n\tLast timestamp ${new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(new Date(0))}\n" +
-				"\tBackpressure 0.17 kB / 4 events\n" +
-				"\tRead throughput 0.255 kB/s or 6 event/s\n" +
-				"\tWrite throughput 0.085 kB/s or 2 event/s\n" +
+				"\tBackpressure 0.966 kB / 4 events\n" +
```

Comment:
Did the `ExampleData` message sizes grow so considerably due to signatures added, or due to the binary -> json change? If the former, the growth seems expected, but if the latter, we might have a problem...

Reply:
I think it's due to the additional fields. The changes in the messages in ExampleData is the following:
+ sequenceNumber (int here but will be long)
+ publisherId (String of 42 chars)
+ previousTimestamp (long)
+ previousSequenceNumber (int here but will be long)
+ signature (String of 132 chars)
- kafkaOffset (int)
- previousKafkaOffset (int)